proto/api/v1/summary_service.proto

```diff
@@ -1,3 +1,4 @@
+// TODO(ROX-24528): This API is deprecated, remove it in 4.7
```

Comment:
In addition to this, definitely add the deprecation note to the rpc definition as well, something like
```
// Deprecated starting 4.5.0 release, scheduled for removal starting 4.7.0.
```